packages/backend/src/services/CatalogService/CatalogService.ts

```diff
@@ -359,4 +359,58 @@ export class CatalogService<
         );
         return { charts: chartAnalytics };
     }
+
+    async getFieldAnalytics(
+        user: SessionUser,
+        projectUuid: string,
```

Comment:
can we also get the `organizationUuid` from user? would remove one `getSummary` query - but all good if not